tests/system/Debug/TimerTest.php

```diff
@@ -121,4 +123,107 @@ public function testReturnsNullGettingElapsedTimeOfNonTimer()
 
         $this->assertNull($timer->getElapsedTime('test1'));
     }
+
+    /**
+     * @timeLimit 1.5
+     */
+    public function testRecordFunctionNoReturn()
+    {
+        $timer       = new Timer();
+        $returnValue = $timer->record('longjohn', static function () { sleep(1); });
```

Comment:
There are several tests to wait for 1 second. If you really need to test 1 second duration,
write one test method for it, and remove other `sleep(1)` in other methods.